__tests__/LottoTest.js

```diff
@@ -15,4 +17,24 @@ describe("로또 클래스 테스트", () => {
   });
 
   // 아래에 추가 테스트 작성 가능
+  test("로또 번호에 숫자가 아닌 입력값이 있으면 예외가 발생한다.", () => {
+    expect(() => {
+      new Lotto(["1", "2", "3", "4", "5", "hi"]);
+    }).toThrow(ERROR_MESSAGES.ONLY_NUMBER);
```

Comment:
```suggestion
    expect(() => new Lotto(["1", "2", "3", "4", "5", "hi"])).toThrow(ERROR_MESSAGES.ONLY_NUMBER);
```
여기서 중괄호를 사용하신 이유가 있으신가요??
if문에서는 중괄호를 생략하셔서 그걸 선호하시는 줄 알았는데 테스트 코드에서는 작성하신 것 같아서 그 이유가 궁금합니다!_!

Reply:
예시로 있었던 LottoTest.js 코드에 이렇게 되어있길래 복붙했습니다 :) 비슷해보이도록 하고 싶어서요 ㅋㅋㅋ

Reply:
테스트 코드짜다보면 위에 작성했던거랑 조금만 바뀌면 되서 복붙 많이 하게 되더라구요🙃